crates/eigenda/src/version.rs

```
@@ -0,0 +1,23 @@
+#[derive(Debug, PartialEq, Copy, Clone)]
+/// Represents the cert version derived from rollup inbox
+/// The version is needed to decode the Cert from serialiezd bytes
+/// Once a valid blob is retrieved, both versions use the identical
+/// logic to derive the rollup channel frame from eigenda blobs
+pub enum CertVersion {
+    /// unknown
+    Unknown,
```

do we really need this? Think this is only typical in protobuf enums, for a reason that I still don't understand. Don't think its needed in rust though. Anything unknown should just return an error when we try to parse it to a correct version

---

it is needed to handle version byte 3,4 which we don't define the cert structure and handling